src/plugins/irc/irc-command.c

```diff
@@ -2148,7 +2148,7 @@ IRC_COMMAND_CALLBACK(dcc)
             {
                 weechat_infolist_new_var_string (item, "plugin_name", weechat_plugin->name);
                 weechat_infolist_new_var_string (item, "plugin_id", ptr_server->name);
-                weechat_infolist_new_var_string (item, "type_string", "file_send");
+                weechat_infolist_new_var_string (item, "type_string", "file_send_passive");
```

Comment:
"file_send_passive" is the same behavior as before here?
Quite confusing for me to read "passive" as it's not, or am I wrong?
So it seems there's no way for WeeChat to initiate both DCC types? I mean for example `/dcc send` (legavy behavior) and `/dcc sendpassive` (new)?